sanic/helpers.py

```diff
@@ -3,6 +3,9 @@
 from importlib import import_module
 from inspect import ismodule
 
+from sanic.signals import Namespace
```

Comment:
Looks like `Namespace` is only imported for Type checking, so has no purpose at runtime.

This import can changed to:
```
if typing.TYPE_CHECKING:
    from sanic.signals import Namespace
```
Then in type declarations, change `Namespace` to `'Namespace'` (string form)